h/static/styles/annotator/inject.scss

```diff
@@ -34,28 +34,45 @@ $base-font-size: 14px;
     width: 6px;
   }
 
-  button {
-    @include box-shadow(none);
-    display: inline-block;
-    font-family: h;
-    border: none;
-    cursor: pointer;
-    height: 100%;
-    width: 35px;
-    font-size: 18px;
-    margin: 0;
-    padding: 0;
-    text-align: center;
-    background: $white !important;
-    color: $gray-light !important;
+  &:hover {
+    .annotator-adder-action button {
+      color: $gray-light !important;
+    }
+  }
 
-    &:active {
-      transition: background-color .25s;
-      background-color: $gray-light !important;
+  .annotator-adder-action {
+    float: left;
+
+    button {
+      @include box-shadow(none);
+      display: inline-block;
+      font-family: h;
+      border: none;
+      cursor: pointer;
+      height: 100%;
+      width: 45px;
+      font-size: 18px;
+      margin: 2px 0px;
+      padding: 0;
+      text-align: center;
+      background: $white !important;
+      color: $gray-dark !important;
+    }
+
+    .adder-label {
```

Comment:
Also on slack: I forgot to talk to you about this: saw your comment on using BEM stying. The card got a little big, are you open to creating a new card for it? Do we want to take this opportunity to refactor the styling in `static/styles/annotator/inject.scss`? It is only 290 lines long.